1 The Physical/Guess Point Level 1 (Stateless)/1 - Red-Green-Refactor Practice/1.1 - Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -10,4 +10,6 @@ export function fizzbuzz(number: number) {
   if (number % 5 === 0) {
     return 'Buzz';
   }
+
+  return '';

```

Comment:
I think it should actually return the given input, instead of an empty string.

Reply:
yeah, this part was missing in the requirements, so I just came up with this.